src/audio/drc/drc.c

```diff
@@ -203,7 +203,7 @@ static int drc_free(struct processing_module *mod)
 {
 	struct drc_comp_data *cd = module_get_private_data(mod);
 
-	comp_info(mod->dev, "drc_free()");
+	comp_dbg(mod->dev, "drc_free()");
```

Comment:
Probably harmless, but seem seem outside the git commit.

Reply:
@kv2019i not sure what you mean by "outside the git commit" but it's important - for the same reason as the comment above - this is actually the final change that eventually fixed pipeline termination crashes with LLEXT - LLEXT modules shouldn't log in their `.free()` and `.reset()` methods, so now these changes - where needed - are always a part of any LLEXT conversion

Reply:
@lyakh if modules should not log in their free/reset, surely this needs to removed then? Having an illegal dbg statement is very misleading. If somebody enables debug logs, she'll get a fail here immedaitely. Better to remove then and even add a comment that don't add log prints here.

Reply:
@kv2019i I agree that switching to debug-level prints isn't perfect for the reasons you mentioned. Removing such logs while adding comments should be a simple and reliable way to fix this. Those logs also don't have a very high value in general - they usually just print the function name, which isn't very informative once the initial code debugging is completed. OTOH I hoped that maybe we manage to resolve this issue. So, yes, if you prefer that, I'm fine to remove those logs completely.

Reply:
Maybe we should move free/reset logs to the function caller place? But not sure if there one single place (would be good) or several of them

Reply:
@abonislawski right, that could be done, but do we really want logs for each `.free()` / `.reset()`? I think removing them would be a better option

Reply:
Not really, Im fine if we delete them